src/GitHub.Api/Tasks/ActionTask.cs

```diff
@@ -221,16 +221,16 @@ class FuncTask<T, TResult> : TaskBase<T, TResult>
         protected Func<bool, T, TResult> Callback { get; }
         protected Func<bool, Exception, T, TResult> CallbackWithException { get; }
 
-        public FuncTask(CancellationToken token, Func<bool, T, TResult> action)
-            : base(token)
+        public FuncTask(CancellationToken token, Func<bool, T, TResult> action, Func<T> getPreviousResult = null)
```

Comment:
I couldn't find any scenarios where `getPreviousResult` is set to a value. Is this a bugfix?

Reply:
Mmm.. spoke too soon